addons/cni-cilium/cilium.yaml

```diff
@@ -193,7 +191,7 @@ data:
   enable-k8s-networkpolicy: "true"
   # Tell the agent to generate and write a CNI configuration file
   write-cni-conf-when-ready: /host/etc/cni/net.d/05-cilium.conflist
-  cni-exclusive: "false"
+  cni-exclusive: "true"
```

Comment:
I'm not sure about this change, I have to double-check where is it coming from

Reply:
I found the following description for this parameter:

```
  # -- Make Cilium take ownership over the `/etc/cni/net.d` directory on the
  # node, renaming all non-Cilium CNI configurations to `*.cilium_bak`.
  # This ensures no Pods can be scheduled using other CNI plugins during Cilium
  # agent downtime.
```

I wasn't able to find why we disabled it in KubeOne, but I stumbled across the following comment in the [KKP codebase](https://github.com/kubermatic/kubermatic/blob/2e30b91169a4f69a13486a58706ea4ce3ec832a6/pkg/cni/cilium/cilium.go#L241-L242):

> // we run Cilium as non-exclusive CNI to allow for Multus use-cases

I believe this might apply to KubeOne too, so I disabled this parameter and commented why is it disabled.